modules/ipi-install-configuring-host-network-interfaces-for-subnets.adoc

```diff
@@ -12,7 +12,9 @@ If you have established different network segments or subnets for remote worker
 
 [IMPORTANT]
 ====
-All control plane nodes must run in the same subnet. When using more than one subnet, you can also configure the Ingress VIP to run on the control plane nodes by using a manifest. See "Configuring network components to run on the control plane" for details.
+You can run control plane nodes in the same subnet or multiple subnets. A multiple subnet environment is useful for reducing the risk of your {product-title} cluster from failing because of a hardware failure or a network outage.
```

Comment:
Have you considered using a snippet for this, since it is repeated?

Reply:
Yeah, good point. I'll do this!